openupgrade_scripts/scripts/project_stock/18.0.1.0/upgrade_analysis.txt

```diff
@@ -1,8 +0,0 @@
----Models in module 'project_stock'---
----Fields in module 'project_stock'---
-project_stock / stock.picking            / project_id (many2one)         : NEW relation: project.project
----XML records in module 'project_stock'---
-NEW ir.embedded.actions: project_stock.project_embedded_action_all_pickings
-NEW ir.embedded.actions: project_stock.project_embedded_action_from_wh
-NEW ir.embedded.actions: project_stock.project_embedded_action_to_wh
-NEW ir.ui.view: project_stock.view_picking_form_inherit_project_stock
```

Comment:
here upgrade_analysis gets confused by the v17 oca addon of the same name being renamed https://github.com/OCA/OpenUpgrade/pull/4960

As I don't think that's harmful, and fixing this would involve having to keep track which renames/merges are relevant for which set of modules, I'd suggest to just do nothing about that